napi.h

```diff
@@ -173,6 +173,10 @@ namespace Napi {
   ///
   /// In the V8 JavaScript engine, a N-API environment approximately corresponds to an Isolate.
   class Env {
+#ifdef NAPI_EXPERIMENTAL
```

Comment:
Same comment as before here as well with respect to `NAPI_VERSION > 5` and the next one as well.